### PR TITLE
Mylin/1630 fix firefox UI

### LINKED ISSUE #1630: UI element rendering issue on Firefox
There are few UI elements rendered in an unexpected way when using Firefox.

- contrast/bias
Screen Shot 2021-08-17 at 9 45 32 AM

- region coordinate referenece (image vs world)
Screen Shot 2021-08-17 at 9 48 55 AM

- animator frame rate spinner (padding not functional?)
Screen Shot 2021-08-17 at 9 52 58 AM

### PR DESCRIPTION
**Pull request by acdo2002**
Fixed #1630 

I have checked with Firefox both on MacOS and UbuntuOS.

### CODE REVIEW

File: src/components/Animator/AnimatorComponent.scss [hunk 1]
```diff
@@ -79,6 +79,7 @@
         }
 
         .playback-numeric-control {
+            margin-right: 10px;
```
Comment (YuHsuan-Hwang) [inline src/components/Animator/AnimatorComponent.scss]: looks like there is an extra space on Chrome after this fix:

File: src/components/RenderConfig/ColormapConfigComponent/ColormapConfigComponent.tsx [hunk 2]
```diff
@@ -63,7 +63,7 @@ export class ColormapConfigComponent extends React.Component<ColormapConfigProps
                     </FormGroup>
                 )}
                 <FormGroup inline={true}>
-                    <Button minimal={true} rightIcon={this.extendBiasContrast ? "double-chevron-up" : "double-chevron-down"} alignText={"right"} small={true} onClick={this.switchExtendBiasContrast}>
+                    <Button minimal={true} className={"bp3-bias-contrast"} rightIcon={this.extendBiasContrast ? "double-chevron-up" : "double-chevron-down"} alignText={"right"} small={true} onClick={this.switchExtendBiasContrast}>
```
Comment (YuHsuan-Hwang) [inline src/components/RenderConfig/ColormapConfigComponent/ColormapConfigComponent.tsx]: please rename it to `bias-contrast-button`

Comment (acdo2002) [reply src/components/RenderConfig/ColormapConfigComponent/ColormapConfigComponent.tsx]: Fixed!

File: src/components/Animator/AnimatorComponent.scss (hunk 1 above)
Comment (acdo2002) [reply src/components/Animator/AnimatorComponent.scss]: I cannot eliminate the space difference between Chrome and Firefox.
So far I only optimize the scss and make sure there is an acceptable space there in both browsers.
Would it be OK or other better solution?

Comment (acdo2002) [reply src/components/Animator/AnimatorComponent.scss]: @YuHsuan-Hwang I follow your suggestion to check and fix the width of "Frame Rate" dropdown, now I hope the scss looks better on both Chrome and Firefox!

Comment (YuHsuan-Hwang) [reply src/components/Animator/AnimatorComponent.scss]: @acdo2002 looks good to me now! Do you think we can change `.bp3-input-group { width: 35px; }` back to 40px?

Comment (acdo2002) [reply src/components/Animator/AnimatorComponent.scss]: Yes, I change it back to 40px and optimize bp3-html-select to 115px.

**Review by YuHsuan-Hwang — APPROVED** [review]

**Review by veggiesaurus — APPROVED** [review]

**Review by kswang1029 — CHANGES_REQUESTED** [review]
@acdo2002 bias/contrast and region dialog fix look good with the three browsers 👍 

Regarding animator part, when the animator widget is resized near the trigger of the responsive layout (eg play buttons with/without text label), the frame rate spinbox may not be rendered properly. This is observed in all three browsers. Please refer to the video. 

https://user-images.githubusercontent.com/20819712/151641056-fbd48ee3-713d-467e-b2a3-9829f669bc5b.mov

**Review by kswang1029 — APPROVED** [review]

**Pull request merged**